sites-available/no-default

```diff
@@ -1,11 +1,13 @@
 # Drop requests for unknown hosts
 #
 # If no default server is defined, nginx will use the first found server.
-# To prevent host header attacks, or other potential problems when an unknown 
-# servername is used in a request, it's recommended to drop the request 
+# To prevent host header attacks, or other potential problems when an unknown
+# servername is used in a request, it's recommended to drop the request
 # returning 444 "no response".
 
 server {
-  listen 80 default_server;
+  listen [::]:80 default_server deferred;
+  listen 80 default_server deferred;
+  server_name _;
```

Comment:
The empty string as value for `server_name` matches requests without a `Host` header field. The intention here is to use a definitely never matching value, for this purpose it is quite common to use `_`. Please consult [nginx documentation](http://nginx.org/en/docs/http/server_names.html) for further details.